src/helpers/documentation.js

```diff
@@ -181,6 +185,13 @@ export const buildDocsContents = (data, rootDir) => {
       : "MISSING METADATA.JSON";
     const articles = {};
 
+    /* if the nodes are coming from /glossary
+    which doesn't have an order specificed in .mdx file
+    sort the nodes alphabetically */
+    if (topic.nodes[0].relativePath.includes("docs/glossary/")) {
```

Comment:
Maybe it'd be feasible to change this to be a folder metadata flag, like `"sortMethod": "alphabetical"` in metadata.json? We've got a number of hardcoded paths already and I'm not really a fan of it (despite writing several of them). I don't think this _needs_ to be changed though.

Reply:
I like that!

Reply:
Updated @vcarl